### PR TITLE
fix: MD037/no-space-in-emphasis

### PR DESCRIPTION
**Pull request by nschonni**
Was mostly just escaping some asterisks that Markdownlint flagged as ambiguous

### CODE REVIEW

File: docs/ios/troubleshooting/mtouch-errors.md [hunk 1]
```diff
@@ -1983,7 +1983,7 @@ This error indicates a bug in Xamarin.iOS. Please file a new issue on [github](h
 
 <a name="MT4140" />
 
-### MT4140: The registrar found a signature mismatch in the method '*.*' - the selector indicates the variadic method takes * parameters, while the managed method has * parameters.
+### MT4140: The registrar found a signature mismatch in the method '\*.\*' - the selector indicates the variadic method takes \* parameters, while the managed method has \* parameters.
```
Comment (nschonni) [inline docs/ios/troubleshooting/mtouch-errors.md:1986]: Not sure if this was rendering incorrectly or was supposed to be a bolded period

File: docs/ios/troubleshooting/mtouch-errors.md [hunk 2]
```diff
@@ -1885,11 +1885,11 @@ This is always a bug in Xamarin.iOS; please file a new issue on [github](https:/
 
 <a name="MT4116" />
 
-### MT4116: Could not register the assembly '*': *
+### MT4116: Could not register the assembly '\*': \*
 
 <a name="MT4117" />
 
-### MT4117: The registrar found a signature mismatch in the method '*.*' - the selector indicates the method takes * parameters, while the managed method has * parameters.
+### MT4117: The registrar found a signature mismatch in the method '\*.\*' - the selector indicates the method takes \* parameters, while the managed method has \* parameters.
```
Comment (nschonni) [inline docs/ios/troubleshooting/mtouch-errors.md:1892]: Not sure if this was rendering incorrectly or was supposed to be a bolded period

Comment (conceptdev) [reply docs/ios/troubleshooting/mtouch-errors.md:1892]: I think your fix is correct - it's intending to indicate a wildcard namespace.class

File: docs/ios/troubleshooting/mtouch-errors.md (hunk 1 above)
Comment (conceptdev) [reply docs/ios/troubleshooting/mtouch-errors.md:1986]: ditto

**Review by conceptdev — APPROVED** [review]
thanks!

**Comment by conceptdev** [comment]
#sign-off

**Pull request merged**